lms/static/js/factories/cohorts_factory.js

```diff
@@ -0,0 +1,35 @@
+;(function (define, undefined) {
+    'use strict';
+    define(['jquery', 'js/groups/views/cohorts', 'js/groups/collections/cohort', 'js/groups/models/course_cohort_settings'],
```

Comment:
I'm wondering why this file is using RequireJS when none of the others are. It is the right long term direction so I suppose it makes sense. But it makes the code a little confusing.